### ISSUE TITLE
When I use Xcode 13.2.1 to build for iOS 12.x, will crash every time

### ISSUE THREAD

**Issue by AZHArvin** [realm/realm-swift#7656]
### How frequently does the bug occur?

All the time

### Description

When I use Xcode 13.2.1 to build for iOS 12.x, will crash every time, but iOS 13.x and later is ok.
pod 'Realm',                                      '~> 10.21.0'
pod 'RealmSwift',                                 '~> 10.21.0'

### Stacktrace & log output

```shell
dyld`__abort_with_payload:
    0x1094842e0 <+0>:  mov    x16, #0x209
    0x1094842e4 <+4>:  svc    #0x80
->  0x1094842e8 <+8>:  b.lo   0x109484300               ; <+32>
    0x1094842ec <+12>: stp    x29, x30, [sp, #-0x10]!
    0x1094842f0 <+16>: mov    x29, sp
    0x1094842f4 <+20>: bl     0x10948371c               ; cerror_nocancel
    0x1094842f8 <+24>: mov    sp, x29
    0x1094842fc <+28>: ldp    x29, x30, [sp], #0x10
    0x109484300 <+32>: ret
```

### Can you reproduce the bug?

Yes, always

### Version

10.21.0

### What SDK flavour are you using?

Local Database only

### Are you using encryption?

No, not using encryption

### Platform OS and version(s)

iOS  12.x

### Build environment

Xcode version: Xcode 13.2.1

**Comment by tgoyne** [comment]
This is a bug in Xcode 13.2.1's concurrency back-deployment which is fixed in 13.3. Until 13.3 is officially released, you need to use 13.1 to support iOS 12.

**Comment by kruti8** [comment]
> This is a bug in Xcode 13.2.1's concurrency back-deployment which is fixed in 13.3. Until 13.3 is officially released, you need to use 13.1 to support iOS 12.

I struggle in realm database and i tried this. And it works for me. Thanks for saving my life. :)
